gammapy/datasets/fermi.py

```diff
@@ -216,6 +218,135 @@ def fetch_fermi_diffuse_background_model(filename='gll_iem_v02.fit'):
 
     return filename
 
+class Fermi3FGLObject(object):
+    """TODO: Doccomments
+    """
+    fermi_cat = fetch_fermi_catalog('3FGL')
+
+    x_bins = np.log10([0.03, 0.1, 0.3, 1, 3, 10, 100])
+
+    y_labels = ['Flux30_100', 'Flux100_300', 'Flux300_1000',
+                'Flux1000_3000', 'Flux3000_10000', 'Flux10000_100000']
+
+    def __init__(self, source_name):
+        self.name_3FGL = source_name
+        self.catalog_index = np.where(self.fermi_cat[1].data['Source_Name'] == source_name)[0][0]
+        self.cat_row = self.fermi_cat[1].data[self.catalog_index]
+        self.ra = self.cat_row['RAJ2000']
+        self.dec = self.cat_row['DEJ2000']
+        self.gal_long = self.cat_row['GLON']
```

Comment:
Please use `glon` and `glat` as variable names here ... this is what we use everywhere in Gammapy (if not, this is what we should use everywhere).